packages/DropdownMenu/stories/examples/DropdownMenuDividersExample.js

```diff
@@ -3,14 +3,8 @@ import DropdownMenu from "../../src";
 
 const DropdownMenuDividersExample = () => {
   return (
-    <DropdownMenu
-      align="bottom"
-      renderTrigger={({ isOpen, handleOpenMenu }) => (
-        <DropdownMenu.Trigger isOpen={isOpen} onOpenMenu={handleOpenMenu}>
-          Trigger
-        </DropdownMenu.Trigger>
-      )}
-    >
+    <DropdownMenu align="bottom">
+      <DropdownMenu.Trigger>Trigger</DropdownMenu.Trigger>
       <DropdownMenu.Item onClick={() => {}}>Item 1</DropdownMenu.Item>
```

Comment:
In an unrelated but taking advantage of this PR.

I don't think onClick should be required on DropDown.Item, 
what bout if you just want to display a `<hr />` ? do we have a `<DropDown.Divider />`?

Reply:
Yes we have a Dropdown.Divider. Currently the `onClick` isn't required either. We have `Dropdown.Item` that have an `onClick` prop, but also `Dropdown.LinkItem` that doens't have an `onClick` prop as it is rendered as an `a` tag